server/delete_function.py

```diff
@@ -0,0 +1,15 @@
+# delete_function
+import os
+import glob
+
+global x
+x = 0
+def delete_all ():
+    global x
+    x = x + 1
+    print (x)
+    if x > 50:
+       files = glob.glob(os.path.join('/home/kajo/Desktop/TTS/TextToSpeech/server/*.mp3'))
```

Comment:
- Why not use the length of files instead of defining a global variable `x` ? **Example**:
```python
def delete_all_tts_files(limit=50):
    files = glob.glob(...
    if len(files) > limit:
        # delete files
```

- Using hard-coded path`/home/kajo/Desktop/TTS/TextToSpeech/server/` will not give the expected result if we're running it from a directory other than the hard-coded one. So we usually solve this problem by getting the current working directory and then joining it with our relative path. **Example**:
```python
# This is the current working directory, it will work regardless of where you're running￼ng the code from
basedir = os.path.abspath(os.path.dirname(__file__))
files = glob.glob(os.path.join(basedir, '*.mp3'))
```

- Final nitpick of the day  😆  when naming functions and variables we should pick a sufficiently descriptive names. Specially with functions, since they contain a lot of logic that can be easily mismatched with other functions logic. Assume you have a lot of things to delete, so you'll have different functions to do that and they can't be all named `delete_all`

Reply:
Thanks ! It's much simpler.